pkg/ponresourcemanager/ponresourcemanager.go

```diff
@@ -124,7 +125,7 @@ const (
 	POOL            = "pool"
 	NUM_OF_PON_INTF = 16
 
-	KVSTORE_RETRY_TIMEOUT = 5
+	KVSTORE_RETRY_TIMEOUT = 5 * time.Second
```

Comment:
Change to following to change data type of constant to duration

time.Duration(5) * time.Second